File: 240116/특정 숫자 도달하기/reaching-specific-number.c
#include <stdio.h>
int main() {
    // 여기에 코드를 작성해주세요.
    int cnt=0;
    int arr[10];
    int sum=0;
    for (int i=0;i<10;i++){

        scanf("%d",&arr[i]);
        if ((arr[i])>=250){
            break;
        }
        cnt+=1;
        sum+=arr[i];
    }
    printf("%d %.1lf",sum,(double)sum/cnt);
    return 0;
}

File: 240305/특정 숫자 도달하기/reaching-specific-number.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc=new Scanner(System.in);
        int a=10;
        int[] arr=new int[10];
        for (int i=0;i<10;i++){
            arr[i]=sc.nextInt();
            if (arr[i]>=250){
                a=i;
                break;
            }
        }
        int sum=0;
        for (int i=0;i<a;i++){
            sum+=arr[i];
        }
        System.out.printf("%d %.1f",sum,(double)sum/a);
    }
}